### PR TITLE
Serialize vol.Maybe

### PR DESCRIPTION
**Pull request by eifinger**
Moved from https://github.com/home-assistant/core/pull/88167

### CODE REVIEW

File: voluptuous_serialize/__init__.py [hunk 1]
```diff
@@ -101,6 +101,11 @@ def convert(schema, *, custom_serializer=None):
             "format": schema.__name__.lower(),
         }
 
+    # vol.Maybe
+    if isinstance(schema, vol.Any):
+        if len(schema.validators) == 2 and schema.validators[0] is None:
+            return convert(schema.validators[1], custom_serializer=custom_serializer)
```
Comment (balloob) [inline voluptuous_serialize/__init__.py:107]: We should not leave out that this was a Maybe.

```suggestion
            result = convert(schema.validators[1], custom_serializer=custom_serializer)
            result["allow_none"] = True
            return result
```

**Comment by eifinger** [comment]
I copied the relvant parts of the flake8 config over from core to make it work with black formatting

**Comment by balloob** [comment]
Thanks! This is such an old repo with not a lot of attention 🙈

**Comment by balloob** [comment]
Released as 2.6.0

**Comment by eifinger** [comment]
Core PR: https://github.com/home-assistant/core/pull/88211

**Pull request merged**